libs/btp/tool-header/src/components/tool-header/tool-header.component.ts

```diff
@@ -144,15 +145,15 @@ export class ToolHeaderComponent extends ToolHeaderComponentClass implements OnD
     /**
      * Whether to show the menu button
      */
-    @Input({ transform: coerceBooleanProperty })
+    @Input({ transform: booleanAttribute })
     showMenuButton: BooleanInput;
```

Comment:
I just figured that this type is not being used. The type of the transform function's parameter is being taken. So this will be `unknown`